includes/wikia/WikiaDataAccess.class.php

```diff
@@ -121,7 +121,7 @@ static function cachePurge( $key ) {
 	 * @param Mixed $result returned result of the callback function which gets data
 	 * @param Integer $cacheTime TTL of memcached data in seconds
 	 * @param Integer $command check description of constants above - USE_CACHE, SKIP_CACHE, REFRESH_CACHE
-	 * 
+	 *
 	 * @author Jakub Olek <jolek@wikia-inc.com>
 	 */
 	static private function setCache( $key, $result, $cacheTime, $command = self::USE_CACHE ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)